constants.go

```diff
@@ -667,7 +667,7 @@ const (
 
 	// ConnectMyComputerRoleNamePrefix is the prefix used for roles prepared for individual users
 	// during the setup of Connect My Computer. The prefix is followed by the name of the cluster
-	// user. See [teleterm.connectmycomputer.RoleSetup].
+	// user. See teleterm.connectmycomputer.RoleSetup.
```

Comment:
That works only if `Features` is in the scope, see https://github.com/gravitational/teleport/pull/28891#discussion_r1261581567

In constants.go, the teleterm package is not imported. Because of this, I'd have to write it out as `[github.com/gravitational/teleport/lib/teleterm/connectmycomputer.RoleSetup]`, which destroys readability IMHO.